### PR TITLE
Readme improvements

### PR DESCRIPTION
**Pull request by andrew-demb**
- 📖 Enable simple to use "copy snippet" feature from GitHub
- 📖 Add "use" to PHP snippets to make clear about FQCN and allow IDE to know classes and methods

### CODE REVIEW

**Comment by andrew-demb** [comment]
CI failure is unrelated

File: README.md [hunk 1]
```diff
@@ -22,11 +22,15 @@ Daily updated from their sources, whether it is ISO or IANA directly, or a maint
 
 To start right away, run the following command in your composer project;
 
-```composer require prinsfrank/standards```
+```bash
```
Comment (PrinsFrank) [inline README.md:25]: I didn't know that the short notation resulted in no copy button, thanks, this makes sense!

File: README.md [hunk 2]
```diff
@@ -172,6 +176,11 @@ All the Alpha2, Alpha3, Numeric and Name values have a corresponding enum in the
 Country group membership can be checked by calling the `isMemberOf` method, supplying the FQN of a class that implements the `GroupInterface`. Several country groups are available: BRICS, EEA, EFTA etc. Countries can also have subdivisions, which can be of several types: countries, provinces, etc.
 
 ```php
+use PrinsFrank\Standards\Country\CountryAlpha2;
+use PrinsFrank\Standards\Country\CountryAlpha3;
+use PrinsFrank\Standards\Country\CountryName;
```
Comment (PrinsFrank) [inline README.md:181]: This namespace is not used in this snippet
```suggestion
```

File: README.md [hunk 3]
```diff
@@ -291,6 +305,10 @@ $valueAlpha2->formatNumber(42.42, LanguageAlpha2::Dutch_Flemish); // '42,42'
 ### CountryAlpha3
 
 ```php
+use PrinsFrank\Standards\Country\CountryAlpha3;
+use PrinsFrank\Standards\Country\Groups\EU;
+use PrinsFrank\Standards\Country\Groups\Brics;
+
```
Comment (PrinsFrank) [inline README.md:311]:
```suggestion
use PrinsFrank\Standards\Language\LanguageAlpha2;

```

File: README.md [hunk 4]
```diff
@@ -324,6 +342,10 @@ $valueAlpha2->formatNumber(42.42, LanguageAlpha2::Dutch_Flemish); // '42,42'
 ### CountryNumeric
 
 ```php
+use PrinsFrank\Standards\Country\CountryNumeric;
+use PrinsFrank\Standards\Country\Groups\EU;
+use PrinsFrank\Standards\Country\Groups\Brics;
+
```
Comment (PrinsFrank) [inline README.md:348]:
```suggestion
use PrinsFrank\Standards\Language\LanguageAlpha2;

```

File: README.md [hunk 5]
```diff
@@ -408,6 +436,8 @@ $valueName->getOfficialAndDeFactoLanguages();    // [LanguageAlpha2::Dutch_Flemi
 Country calling codes are quite straight forward. One country can have multiple country calling codes though, And one country calling code can span multiple countries. That's why the `forCountry` and `getCountriesAlpha2` both return an array of country calling codes/countries and not a single item.
 
 ```php
+use PrinsFrank\Standards\CountryCallingCode\CountryCallingCode;
```
Comment (PrinsFrank) [inline README.md:439]:
```suggestion
use PrinsFrank\Standards\Country\CountryAlpha2;
use PrinsFrank\Standards\CountryCallingCode\CountryCallingCode;
```

File: README.md [hunk 6]
```diff
@@ -433,6 +463,11 @@ public function foo(CountryCallingCode $countryCallingCode) {} // Use spec as ty
 All the Alpha3, Numeric and Name values have a corresponding enum in the other currency enums. These can be converted using their corresponding methods (toCurrencyAlpha3, etc...). A fourth enum is available that maps all currencies to a currency symbol, that can be accessed by calling the 'getSymbol' method.
 
 ```php
+use PrinsFrank\Standards\Currency\CurrencyAlpha3;
```
Comment (PrinsFrank) [inline README.md:466]:
```suggestion
use PrinsFrank\Standards\Country\CountryAlpha2;
use PrinsFrank\Standards\Currency\CurrencyAlpha3;
```

File: README.md [hunk 7]
```diff
@@ -433,6 +463,11 @@ public function foo(CountryCallingCode $countryCallingCode) {} // Use spec as ty
...
+use PrinsFrank\Standards\Currency\CurrencyName;
+use PrinsFrank\Standards\Currency\CurrencyNumeric;
+use PrinsFrank\Standards\Currency\CurrencySymbol;
+
```
Comment (PrinsFrank) [inline README.md:470]:
```suggestion
use PrinsFrank\Standards\Language\LanguageAlpha2;

```

File: README.md [hunk 8]
```diff
@@ -549,6 +590,8 @@ $currencyNumeric->format(42.42, CountryAlpha2::Netherlands, LanguageAlpha2::Dutc
 ### CurrencySymbol
 
 ```php
+use PrinsFrank\Standards\Currency\CurrencySymbol;
```
Comment (PrinsFrank) [inline README.md:593]:
```suggestion
use PrinsFrank\Standards\Currency\CurrencyAlpha3;
use PrinsFrank\Standards\Currency\CurrencySymbol;
```

File: README.md [hunk 9]
```diff
@@ -647,6 +704,11 @@ There are four language code specifications:
 As you see, the Bibliographic and Terminology specifications have an identical number of languages, so there is a one-to-one relation between these.
 
 ```php
+use PrinsFrank\Standards\Language\LanguageAlpha2;
+use PrinsFrank\Standards\Language\LanguageAlpha3Terminology;
+use PrinsFrank\Standards\Language\LanguageAlpha3Extensive;
+use PrinsFrank\Standards\Language\LanguageAlpha2;
```
Comment (PrinsFrank) [inline README.md:710]:
```suggestion
```

This use is here twice

File: README.md [hunk 10]
```diff
@@ -725,6 +796,9 @@ $valueName = $valueAlpha3->name;                     // 'Dutch'
 ### At a glance
 
 ```php
+use PrinsFrank\Standards\Language\LanguageTag;
```
Comment (PrinsFrank) [inline README.md:799]:
```suggestion
use PrinsFrank\Standards\Country\CountryAlpha2;
use PrinsFrank\Standards\LanguageTag\LanguageTag;
```

File: README.md [hunk 11]
```diff
@@ -725,6 +796,9 @@ $valueName = $valueAlpha3->name;                     // 'Dutch'
...
+use PrinsFrank\Standards\Language\LanguageAlpha2;
```
Comment (PrinsFrank) [inline README.md:800]:
```suggestion
use PrinsFrank\Standards\Language\LanguageAlpha2;
use PrinsFrank\Standards\LanguageTag\LanguageTagVariant;
```

File: README.md [hunk 12]
```diff
@@ -792,6 +874,10 @@ public function foo(NationalCallPrefix $nationalCallPrefix) {} // Use spec as ty
 ### At a glance
 
 ```php
+use PrinsFrank\Standards\Scripts\ScriptAlias;
+use PrinsFrank\Standards\Scripts\ScriptCode;
+use PrinsFrank\Standards\Scripts\ScriptName;
```
Comment (PrinsFrank) [inline README.md:879]:
```suggestion
use PrinsFrank\Standards\Scripts\ScriptName;
use PrinsFrank\Standards\Scripts\ScriptNumber;
```

**Review by PrinsFrank — CHANGES_REQUESTED** [review]
Hi, thanks for the contribution! I've walked through all of the changes and checked namespaces, there are some that are not complete or incorrect. Let me know if you need help!

File: README.md (hunk 12 above)
Comment (andrew-demb) [reply README.md:884]: ++

File: README.md (hunk 11 above)
Comment (andrew-demb) [reply README.md:804]: ++

File: README.md (hunk 10 above)
Comment (andrew-demb) [reply README.md:803]: ++

File: README.md (hunk 9 above)
Comment (andrew-demb) [reply README.md]: ++

File: README.md (hunk 8 above)
Comment (andrew-demb) [reply README.md:597]: ++

File: README.md (hunk 7 above)
Comment (andrew-demb) [reply README.md:473]: ++

File: README.md (hunk 6 above)
Comment (andrew-demb) [reply README.md:469]: ++

File: README.md (hunk 5 above)
Comment (andrew-demb) [reply README.md:441]: ++

File: README.md (hunk 4 above)
Comment (andrew-demb) [reply README.md:349]: ++

File: README.md (hunk 3 above)
Comment (andrew-demb) [reply README.md:311]: ++

File: README.md (hunk 2 above)
Comment (andrew-demb) [reply README.md]: ++

**Comment by PrinsFrank** [comment]
Thank you!!!

**Pull request merged**